yarn.lock

```diff
@@ -6681,7 +6681,7 @@ keytar@7.2.0:
   version "7.2.0"
   resolved "https://registry.yarnpkg.com/keytar/-/keytar-7.2.0.tgz#4db2bec4f9700743ffd9eda22eebb658965c8440"
   integrity sha512-ECSaWvoLKI5SI0pGpZQeUV1/lpBYfkaxvoSp3zkiPOz05VavwSfLi8DdEaa9N2ekQZv3Chy+o7aP6n9mairBgw==
-  dependencies:
+    dependencies:
```

Comment:
What is this indent doing here?

Reply:
Mostly revealing that I have my git settings set to ignore whitespace changes :-).